active/000-buisness-attributes/000-business-attributes.md

```diff
@@ -0,0 +1,258 @@
+- Start Date: 2023-10-19
+- RFC PR: 000-business-attributes.md
+- Discussion Issue: 
+- Implementation PR(s):
+
+# Business Attributes
+
+## Summary
+
+A Business attribute is a centrally managed logical field that represents a unique schema field entity. This common construct is global in nature, i.e. it is not bound to a project or application implementation. Instead, its identity exists in representing the same field across various datasets owned by various different projects and applications. Projects or applications use the Business attribute to model a column in a dataset and inherit information about it such as definitions, data type, data quality rules/assertions, tags, glossary terms etc from the global definition. Data architects can use the concept of the business attribute to validate whether applications are conformant with the applicable metadata defined for the business attribute. By abstracting common business metadata into a logical model, different personas with appropriate business knowledge can define pertinent details, like rich definition, business use for the attribute, classification (i.e. PII, sensitive, shareable etc.), specific data rules that govern the attribute, connection to glossary terms.
+
+## Motivation
+
+* Improve the metadata definition, consistency, and meaning for commonly used attributes across applications / data systems in the organisation.
+* Business users/consumers can easily discover datasets they are looking for  with the help of business attributes
+* Add the ability to define data rules to govern the data in this attribute across applications
+
+
+## Requirements
+
+The primary personas involved in this are the Data Architect, Dataset Owner and Business User personas
+
+### Data Architect
+
+#### Must Haves
+1. Ability to define a business attribute, and associate a description, datatype, glossary terms, tags and DQ rules among other aspects of a general field
+2. Ability to update a business attribute record centrally
+3. Ability to track any discrepancies between business attributes and associated fields
+
+#### Good-to-Haves
+1. Approval workflow for creation/updation
+
+### Dataset Owner
+
+#### Must Haves
+1. Ability to inherit all attributes from a business attribute to a field by attaching the business attribute to the field
+1. Ability to inherit all updates from a business attribute to a field
+
+#### Good-to-Haves
+
+1. Ability to intelligently inherit attributes from business attributes to fields in datasets via system auto-assignment
+
+### Business User
+
+#### Must Haves
+1. Ability to search for fields using business description/tags/glossary attached to business attribute
```

Comment:
Is the expectation that this expanded search should be the default universal search experience on the main search bar?
How should ranking work when you have a match on a business description through a business attribute attached to a field versus a match on a field level description .. or a match on the table description?

Reply:
Our ranking strategy prioritizes field-level descriptions, followed by descriptions of business attributes. The primary objective is to ensure that the corresponding dataset is displayed as long as there is a match with the business attribute description.